wolfcrypt/src/asn.c

```diff
@@ -955,7 +957,15 @@ int GetInt(mp_int* mpi, const byte* input, word32* inOutIdx, word32 maxIdx)
         return ASN_GETINT_E;
     }
 
+#ifdef HAVE_WOLF_BIGINT
```

Comment:
This code allows the removal of the GetRsaInt that used to be here. I agree there is probably some extra work here for certain algorithms. I'll make a cleanup note to investigate this.